<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>990</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>990</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face Regconition App</string>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="startFuncButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>430</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    border-image: url(image/button.png) 0 0 0 0 stretch stretch;
}
QPushButton::pressed
{
    border-image: url(image/pressButton.png) 0 0 0 0 stretch stretch;
}</string>
   </property>
   <property name="text">
    <string>Bật nguồn camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profileButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>210</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Danh sách người dùng</string>
   </property>
  </widget>
  <widget class="QLabel" name="outLine">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>450</width>
     <height>510</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 2px;
border-radius: 10px;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="outLine2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>180</y>
     <width>430</width>
     <height>330</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="imageScreen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>510</width>
     <height>501</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="historyButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>100</y>
     <width>210</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lịch sử đăng nhập</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="historyDisplay">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>310</y>
     <width>270</width>
     <height>190</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>268</width>
      <height>188</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="userName">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Họ và tên:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userSex">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Giới tính:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userDob">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>240</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ngày sinh:</string>
   </property>
  </widget>
  <widget class="QLabel" name="userImage">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>190</y>
     <width>130</width>
     <height>110</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>270</y>
     <width>270</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lịch sử 4 lần đăng nhập gần nhất:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>310</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Đăng nhập</string>
   </property>
  </widget>
  <widget class="QLabel" name="mask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>510</width>
     <height>501</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>360</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Quicksand</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>imageScreen</zorder>
  <zorder>outLine</zorder>
  <zorder>outLine2</zorder>
  <zorder>startFuncButton</zorder>
  <zorder>profileButton</zorder>
  <zorder>historyButton</zorder>
  <zorder>historyDisplay</zorder>
  <zorder>userImage</zorder>
  <zorder>loginButton</zorder>
  <zorder>mask</zorder>
  <zorder>editButton</zorder>
  <zorder>userName</zorder>
  <zorder>userSex</zorder>
  <zorder>userDob</zorder>
  <zorder>label_4</zorder>
 </widget>
 <resources>
  <include location="../../Face-Recogntion-PyQt/Face_Detection_PyQt_Final/resource.qrc"/>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
